.gitignore

```diff
@@ -22,3 +22,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+.vscode/settings.json
+src/App.jsx
```

Comment:
I don't believe App.jsx should be in the .gitignore file. Am I missing something?